lib/chef/provider/execute.rb

```diff
@@ -78,6 +78,18 @@ def sensitive?
         !!new_resource.sensitive
       end
 
+      def live_stream?
+        !!new_resource.live_stream
```

Comment:
Seems like this should be true if either:
* `Chef::Config[:stream_execute_output]` is true
* `new_resource.live_stream` is true

Right? I'm thinking that `Chef::Config[:stream_execute_output]` means "stream every execute resource (except if marked sensitive)" and `new_resource.live_stream` means "stream this one execute resource, even if streaming for execute is disabled globally". Is that right?